wallet-new/src/Cardano/Wallet/Orphans/Arbitrary.hs

```diff
@@ -28,5 +29,20 @@ instance Arbitrary CPaperVendWalletRedeem
 instance Arbitrary CInitialized
 instance Arbitrary NewBatchPayment
 
+instance Arbitrary CCoin where
+    arbitrary = CCoin <$> arbitrary
+
+instance Arbitrary CUpdateInfo where
+    arbitrary = CUpdateInfo <$> arbitrary
+                            <*> arbitrary
+                            <*> arbitrary
+                            <*> arbitrary
+                            <*> arbitrary
+                            <*> arbitrary
+                            <*> arbitrary
+                            <*> arbitrary
+
 instance Arbitrary WalletStateSnapshot where
-    arbitrary = WalletStateSnapshot <$> pure def
+    arbitrary = do
+        v <- arbitrary
+        WalletStateSnapshot <$> pure (def {_wsReadyUpdates = [v]})
```

Comment:
Nice! This fixes the swagger spec problem?

Why not generate a list of `v`s instead of just a single `v`?

Reply:
> This fixes the swagger spec problem?

It seems that Swagger does not like an empty `[]` in that case, so we have to fill it up. There is a comment in [ `Example.hs`](https://github.com/input-output-hk/cardano-sl/blob/8c2c240406957c9fb688cd6a0d7bf569e2f8d48e/wallet-new/server/Cardano/Wallet/API/V1/Swagger/Example.hs#L111-L115) about how to deal w/ such an issue 

> Why not generate a list of vs instead of just a single v?

At the moment the only place we do use an arbitrary instance of `WalletStateSnapshot` is for generating an example for Swaggers documentation. And a single item is all what we need in that example. If we might need a "real" arbitrary instance without using [`Default` of `WalletStorage`](https://github.com/input-output-hk/cardano-sl/blob/8c2c240406957c9fb688cd6a0d7bf569e2f8d48e/wallet/src/Pos/Wallet/Web/State/Storage.hs#L228-L241) (which provides more `mempty` then any arbitrary values), then we can change it. But for now I would suggest just to add just a single item in that list of `wsReadyUpdates`.

Reply:
Sounds good :+1: